compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -1008,41 +1009,40 @@ impl<'a> Parser<'a> {
             }
             token::Literal(token::Lit { kind: token::Float, symbol, suffix }) => {
                 Ok(match self.break_up_float(symbol, self.token.span) {
-                    // 1e2
```

Comment:
FYI I copied these comments from another location in this file. If you grep for "1.2 | 1.2e3" for example, you'll find a bunch of locations in this file, not all of them involving `DestructuredFloat`.